src/utils/graph_prune_utils.py

```diff
@@ -118,13 +115,18 @@ def _prune_edges_by_minhash_lsh_helper(num_nodes,
 
         # Transform the adjacent nodes to their signatures
         results = minhash.apply(adjacent_reps, adjacent_meta)
+        results_metas = [result.meta for result in results]
```

Comment:
Break apart these parts for better readability (for loop should be just a few lines long)